core/test_network/src/lib.rs

```diff
@@ -61,6 +61,15 @@ pub fn get_key_pair() -> KeyPair {
     ).unwrap()
 }
 
+/// Get hex string represented genesis_config for ./genesis.json that is tied to hardcoded keys and chain_id
+pub fn get_genesis_signature() -> GenesisSignature {
+    let manifest_dir = Path::new(env!("CARGO_MANIFEST_DIR"));
+    let genesis = RawGenesisBlock::from_path(manifest_dir.join("../../configs/swarm/genesis.json"))
```

Comment:
Shouldn't genesis signature be computed from a genesis block that is generated within tests? I remember that different tests generate it dynamically depending on their needs.